agent/engine/docker_image_manager.go

```diff
@@ -273,6 +275,12 @@ func (imageManager *dockerImageManager) isImageOldEnough(imageState *image.Image
 	return ageOfImage > imageManager.minimumAgeBeforeDeletion
 }
 
+//TODO: change createdTime to pullAtTime when docker support it in the future
+func (imageManager *dockerImageManager) nonECSImageOldEnough(NonECSImage ImageWithSizeID) bool {
+	ageOfImage := time.Now().Sub(NonECSImage.createdTime)
+	return ageOfImage >= imageManager.nonECSMinimumAgeBeforeDeletion
```

Comment:
It does not matter very much if its `>=` or `>`. just to be consistent with `isImageOldEnough`, we can make it just `>`